From 9cffb960562146da4398d7bf7c9a879e7c27418e Mon Sep 17 00:00:00 2001
From: Arulpandiyan Vadivel <arulpandiyan_vadivel@mentor.com>
Date: Thu, 9 May 2019 17:47:36 +0530
Subject: [PATCH 2/2] MLK-21603 rpmsg: imx_rpmsg: Fix to stop autoload modules

commit 4cce7ffbb7c3065b8c91450581c772e3e2aed96a from
https://source.codeaurora.org/external/imx/linux-imx.git

This change stops autoloading imx_rpmsg_tty and imx_rpmsg_pingpong
modules, as it is only meant for testing / demo purposes.

Signed-off-by: Arulpandiyan Vadivel <arulpandiyan_vadivel@mentor.com>
Reviewed-by: Leonard Crestez <leonard.crestez@nxp.com>
Signed-off-by: Xulin Sun <xulin.sun@windriver.com>
---
 drivers/rpmsg/imx_rpmsg_pingpong.c | 1 -
 drivers/rpmsg/imx_rpmsg_tty.c      | 1 -
 2 files changed, 2 deletions(-)

diff --git a/drivers/rpmsg/imx_rpmsg_pingpong.c b/drivers/rpmsg/imx_rpmsg_pingpong.c
index 3aa545fcfe14..cf02bb957434 100644
--- a/drivers/rpmsg/imx_rpmsg_pingpong.c
+++ b/drivers/rpmsg/imx_rpmsg_pingpong.c
@@ -80,7 +80,6 @@ static struct rpmsg_device_id rpmsg_driver_pingpong_id_table[] = {
 	{ .name	= "rpmsg-openamp-demo-channel-1" },
 	{ },
 };
-MODULE_DEVICE_TABLE(rpmsg, rpmsg_driver_pingpong_id_table);
 
 static struct rpmsg_driver rpmsg_pingpong_driver = {
 	.drv.name	= KBUILD_MODNAME,
diff --git a/drivers/rpmsg/imx_rpmsg_tty.c b/drivers/rpmsg/imx_rpmsg_tty.c
index 8228c0da287a..6d9e44e4b060 100644
--- a/drivers/rpmsg/imx_rpmsg_tty.c
+++ b/drivers/rpmsg/imx_rpmsg_tty.c
@@ -224,7 +224,6 @@ static struct rpmsg_device_id rpmsg_driver_tty_id_table[] = {
 	{ .name = "rpmsg-openamp-demo-channel" },
 	{ },
 };
-MODULE_DEVICE_TABLE(rpmsg, rpmsg_driver_tty_id_table);
 
 static struct rpmsg_driver rpmsg_tty_driver = {
 	.drv.name	= KBUILD_MODNAME,
-- 
2.17.1

